src/views/search/summary/FactualConsistencyBadge.tsx

```diff
@@ -26,23 +26,16 @@ const getConfidenceLevel = (score: number): ConfidenceLevel => {
 export const FactualConsistencyBadge = ({ score, summaryShowFcsBadge }: Props) => {
     let badge;
     const fcsLDocsLink = "https://docs.vectara.com/docs/api-reference/search-apis/search?#factual-consistency-score"
+    console.log("score", score);
```

Comment:
nit: is that necessary or we can remove it?

Reply:
removed